ghostly.py

```diff
@@ -97,6 +97,16 @@ def load(self, url):
         self.browser.get(url)
 
 
+    def navigate(self, navigation):
+        """
+        Navigate with browser history
+        http://selenium-python.readthedocs.org/navigating.html#navigation-history-and-location
+        TODO: handle all the methods available!
+        back, forward, refresh, to(navigation)
+        """
+        self.browser.back()
```

Comment:
We're going to have to get tricky here if you want to overload the `navigate` function to support the multiple methods available on `browser`.

I'm thinking that we'll probably want to create separate back / forward / refresh commands eventually and hide this navigate command.

Anyway, can be implemented like this:

```
def navigate(self, navigation):
    f = getattr(self.browser, navigation)
    f()
```